mast_aladin_lite/aida.py

```diff
@@ -40,3 +41,66 @@ def set_viewport(self, center):
             )
 
         self.app.target = center
+
+    def get_viewport(
+        self,
+        sky_or_pixel: str | None = "sky",
+        image_label: str | None = None
+    ) -> dict[str, Any]:
+        """
+        Gets the current viewport of the viewer.
+        Presently, returns a dictionary containing the viewer's `center`,
+        given as `~astropy.coordinates.SkyCoord` objects, and the viewer's
+        `fov`, given as an `~astropy.units.Angle` object.
```

Comment:
Let's keep the top line up here to one sentence, since the `Returns` section below covers it:
```suggestion
        Get the viewport center and field of view.
```